release-notes/10.0/preview/preview1/containers.md

```diff
@@ -0,0 +1,45 @@
+# Container image updates in .NET 10 Preview 1 - Release Notes
+
+.NET 10 Preview 1 includes the following updates for container images:
+
+- [New OS: Debian 13 "Trixie"](#new-os-debian-13-trixie)
+- [Ubuntu is now the default OS](#ubuntu-is-now-the-default-os)
```

Comment:
Thoughts on listing this first?  I realize it is a short list but this one has the potential to be the most impactful.  It feels like it would be best to list these in order of most impactful.